app/src/main/java/org/fossasia/openevent/app/model/EventDetails.java

```diff
@@ -1,362 +1,360 @@
     @SerializedName("organizer_name")
-    private String mOrganizerName;
+    private String organizerName;
     @SerializedName("placeholder_url")
-    private String mPlaceholderUrl;
+    private String placeholderUrl;
     @SerializedName("privacy")
-    private String mPrivacy;
+    private String privacy;
     @SerializedName("schedule_published_on")
-    private Object mSchedulePublishedOn;
+    private String schedulePublishedOn;
     @SerializedName("searchable_location_name")
-    private String mSearchableLocationName;
+    private String searchableLocationName;
     @SerializedName("social_links")
-    private List<Object> mSocialLinks;
+    private List<SocialLink> socialLinks;
     @SerializedName("start_time")
-    private String mStartTime;
+    private String startTime;
     @SerializedName("state")
-    private String mState;
+    private String state;
     @SerializedName("sub_topic")
-    private String mSubTopic;
+    private String subTopic;
     @SerializedName("thumbnail")
-    private String mThumbnail;
+    private String thumbnail;
     @SerializedName("ticket_url")
-    private String mTicketUrl;
-    @SerializedName("tickets")
-    private List<Ticket> mTickets;
+    private String ticketUrl;
+    @SerializedName("TICKETS")
+    private List<Ticket> tickets;
     @SerializedName("timezone")
-    private String mTimezone;
+    private String timezone;
     @SerializedName("topic")
-    private String mTopic;
+    private String topic;
     @SerializedName("type")
-    private String mType;
+    private String type;
     @SerializedName("version")
-    private Version mVersion;
+    private Version version;
 
     public String getBackgroundImage() {
-        return mBackgroundImage;
+        return backgroundImage;
     }
 
     public void setBackgroundImage(String backgroundImage) {
-        mBackgroundImage = backgroundImage;
+        this.backgroundImage = backgroundImage;
     }
 
     public Object getCallForPapers() {
-        return mCallForPapers;
+        return callForPapers;
     }
 
     public void setCallForPapers(Object callForPapers) {
-        mCallForPapers = callForPapers;
+        this.callForPapers = callForPapers;
     }
 
     public String getCodeOfConduct() {
-        return mCodeOfConduct;
+        return codeOfConduct;
     }
 
     public void setCodeOfConduct(String codeOfConduct) {
-        mCodeOfConduct = codeOfConduct;
+        this.codeOfConduct = codeOfConduct;
     }
 
     public Copyright getCopyright() {
-        return mCopyright;
+        return copyright;
     }
 
     public void setCopyright(Copyright copyright) {
-        mCopyright = copyright;
+        this.copyright = copyright;
     }
 
     public String getDescription() {
-        return mDescription;
+        return description;
     }
 
     public void setDescription(String description) {
-        mDescription = description;
+        this.description = description;
     }
 
     public Object getEmail() {
```

Comment:
That's a bug in JSPNSchema2POJO which was originally in the code, I replaced it almost everywhere, this one I must have forgot. Thanks